backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/radiobutton/BpkRadioButton.kt

```diff
@@ -122,7 +121,7 @@ private fun BpkRadioButtonImpl(
       colors = RadioButtonDefaults.colors(
         selectedColor = BpkTheme.colors.primary,
         unselectedColor = BpkTheme.colors.textSecondary,
-        disabledColor = BpkColor.SkyGrayTint04,
+        disabledColor = BpkTheme.colors.textDisabled,
```

Comment:
is that intended? that's the radiobutton colour that doesn't seem to change colour in night mode based on figma